definitions/scenarios/upgrade_to_satellite_6_16.rb

```diff
@@ -29,6 +29,7 @@ def compose
       add_steps(find_checks(:pre_upgrade))
       add_step(Checks::Disk::AvailableSpacePostgresql13)
       add_step(Checks::Repositories::Validate.new(:version => '6.16'))
+      add_step(Checks::CheckOrganizationContentAccessMode)
```

Comment:
@ehelms I don't see a corresponding upgrade scenario for Katello 4.12 -- the only upstream upgrade scenario is Katello Nightly.

Is this what you were referring to when you previously said we don't do any branding here?

Should I add this check to the Nightly upgrade scenario as well, or should I leave it Satellite only and remove the branding of instance_version string?

Reply:
Fair question - as the Katello scenario is nightly only and I do not think our docs mention using foreman-maintain upgrade then being Satellite specific about this is OK. Please verify the docs part though.

Reply:
I was just looking at the upgrade docs, and indeed they don't mention foreman-maintain. The upstream upgrade procedure is just dnf update followed by an installer run.

Reply:
Thanks. So upstream docs at https://docs.theforeman.org/nightly/Upgrading_Project/index-katello.html use foreman-maintain for starting/stopping services only; the upgrade itself is still done manually

I've pushed an update that removes references to Katello and always uses downstream names